ESPController/src/main.cpp

```diff
@@ -3057,7 +3067,7 @@ ESP32 Chip model = %u, Rev %u, Cores=%u, Features=%u)RAW",
            GIT_VERSION, COMPILE_DATE_TIME,
            chip_info.model, chip_info.revision, chip_info.cores, chip_info.features);
 
-  esp_bt_controller_disable();
+  ESP_ERROR_CHECK_WITHOUT_ABORT(esp_bt_controller_disable());
```

Comment:
technically this line shouldn't be necessary since BT won't be running at this point.

Reply:
I must have added this in for a reason - didn't the Arduino libs start all this on power up?  I used to remember it also starting the wifi etc.